stream-chat-android/src/main/java/com/getstream/sdk/chat/view/dialog/MessageMoreActionDialog.kt

```diff
@@ -49,47 +50,48 @@ internal class MessageMoreActionDialog(
     }
 
     private fun setupMessageActions() {
-        setContentView(R.layout.stream_dialog_message_moreaction)
+        val binding = StreamDialogMessageMoreactionBinding.inflate(context.inflater)
+        setContentView(binding.root)
```

Comment:
Using `setContentView(View)` instead of `setContentView(@LayoutRes Int)` somehow changed how the layout was positioned. Previously, it was positioned at the bottom the screen correctly, but this change moved some of its contents to the top of the screen. The changes in the layout XML fix this.